bouncerscript/test/test_task.py

```diff
@@ -80,4 +77,4 @@ def test_get_supported_actions():
 
 
 def test_validate_task_schema(submission_context, schema="submission"):
```

Comment:
Improvement: Check the alias schema too, thanks to `@pytest.mark.parametrize`. Not blocking.

Reply:
Slightly more complicated I think. The `schema` is 'submission' there because we're operating with `submission_context`. I'd happily add the aliases_context too along with its schema to check but as far as I know we can't feed fixtures (with scope for functions) within pytest.parametrize. Is there a way to do this?

Reply:
Feel free to reply here, I'll address in my upcoming PR for bug 1445684.